### DISCUSSION TITLE
Could I be added to the project?

### DISCUSSION THREAD

**Discussion by psprint** [zdharma-continuum/zinit#459] [General]
Hi,
I'm the author of zinit. I've once removed everything on github, to see that zdharma-continuum appeared to preserve and grow my work. Thank you for this. That said, I have 10 unmerged pull requests from last ~5 months. I see that the changes might require much effort to gain certainty that they're save to merge. This makes me feel that I should be able to take own responsibility for them. So I thought that I could be maybe added to zinit project as a contributor. What do you think?

**Comment by alichtman** [comment]
Hey @psprint,

@vladdoster and @pschmitt and I have talked about it. 

Given your history of `rm -rf`ing everything (_twice_), we have decided not to grant you any privileges on the `zdharma-continuum` org. We spent a lot of time undoing the great deletion in Nov 2021. This may be in the past, but we still believe it would be too risky for the project going forward to give you write privileges. Feel free to fork it and do whatever you'd like on your fork.

As to your recent feature proposals: we, for the most part, do not understand the use-cases you are trying to solve. We (and esp @vladdoster) have been focusing on stability and resolving bugs, not adding features that we will have to maintain in the future. Your contributions are always welcome. We know it may be frustrating that you do not get prompt reviews, but that is sort of just how open source is. We all do this in our free time.

**Reply by psprint** [reply]
Ok, i understand, however some PRs are easy to grasp, e.g. #333. Could they be merged quicker?

**Reply by alichtman** [reply]
Unfortunately, I personally don't have too much time to invest in the project.

**Comment by vladdoster** [comment]
Locking this thread for posterity.
